src/connectedk8s/azext_connectedk8s/commands.py

```diff
@@ -22,7 +22,7 @@ def load_command_table(self, _):
     )
     with self.command_group('connectedk8s', connectedk8s_sdk, client_factory=cf_connected_cluster) as g:
         g.custom_command('connect', 'create_connectedk8s', supports_no_wait=True)
-        g.custom_command('update', 'update_agents')
+        g.custom_command('update', 'update_agents_or_resource', supports_no_wait=True)
```

Comment:
what is impact of `supports_no_wait`?

Reply:
'supports_no_wait' is used for async operations. its not related to skip prompt. If the user passes '--no-wait', the CLI command will not wait for async operation to finish and exit